### ISSUE TITLE
Quoted-printable strings in subject not decoded correctly

### ISSUE THREAD

**Issue by wiktor-k** [jhermsmeier/node-envelope#18]
Hello,
I just found your library and it seems to be simple and produces really clean objects but I found one issue when parsing Subject headers that are quoted-printable.

My `email.eml` file contains this line: `Subject: =?UTF-8?Q?a=c5=82o=c5=9b-?=`

Running it through envelope with this code:
```js
var fs = require( 'fs' )
var Envelope = require( 'envelope' )
var data = fs.readFileSync( './email.eml' )

var email = new Envelope( data )
console.log(email.header.subject);

// libqp for comparison
var libqp = require('libqp');
var s = libqp.decode('a=c5=82o=c5=9b-').toString('utf8');
console.log(s);
```

Produces the following output:
```
a=c5�o=c5=9b- // badly decoded string from envelope
ałoś-          // libqp parses this correctly
```

I've also opened the file in an e-mail client and surely enough the subject was displayed correctly.

I'm using version 2.0.0 (previous version 1.0.9 did not touch quoted-printable subject at all I think).

**Comment by jhermsmeier** [comment]
Hi @wiktor-k – thanks for bringing this up! I might have recently broken this – which version are you using?

**Comment by wiktor-k** [comment]
I just edited the original but I tested with 2.0.0 that as far as I can see was released minutes ago :)

**Comment by jhermsmeier** [comment]
Can confirm, this is a bug in [mime-lib](https://github.com/jhermsmeier/node-mime-lib) – will try and fix soon!

**Comment by jhermsmeier** [comment]
@wiktor-k found the buggery, and fixed in `envelope@2.0.1`

**Comment by wiktor-k** [comment]
Thanks a lot for this quick fix!
